html/about.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>About</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+</head>
+
+    <body>
+        {{template "navbar"}}
+
+        <div class="container">
+            {{if ne .ErrorMessage ""}}
+            <div class="alert alert-danger">
+                {{.ErrorMessage}}
+            </div>
+            {{end}}
+
+            <div class="container pt-5 pl-0">
+                <h2>About<br></h2>
```

Comment:
This can still say "Installed Packages" as this is the Installed Packages section of the About page.

Soon we can also add more details about device serial numbers and hardware details.